docs/reference/search/suggesters/phrase-suggest.asciidoc

```diff
@@ -141,21 +141,21 @@ The response contains suggestions scored by the most likely spell correction fir
 
 [horizontal]
 `field`::
-    the name of the field used to do n-gram lookups for the
+    The name of the field used to do n-gram lookups for the
     language model, the suggester will use this field to gain statistics to
     score corrections. This field is mandatory.
 
 `gram_size`::
-    sets max size of the n-grams (shingles) in the `field`.
-    If the field doesn't contain n-grams (shingles) this should be omitted
+    Sets max size of the n-grams (shingles) in the `field`.
+    If the field doesn't contain n-grams (shingles), this should be omitted
     or set to `1`. Note that Elasticsearch tries to detect the gram size
-    based on the specified `field`. If the field uses a `shingle` filter the
+    based on the specified `field`. If the field uses a `shingle` filter, the
     `gram_size` is set to the `max_shingle_size` if not explicitly set.
 
 `real_word_error_likelihood`::
-    the likelihood of a term being a
+    The likelihood of a term being a
     misspelled even if the term exists in the dictionary. The default is
-    `0.95` corresponding to 5% of the real words are misspelled.
+    `0.95`, meaning 5% of the real words are misspelled.
```

Comment:
I was a bit unsure about the accuracy of this edit.

Reply:
++, the change looks good